### DISCUSSION TITLE
Font Size for Stimulus

### DISCUSSION THREAD

**Discussion by BobRoss048** [jspsych/jsPsych#1178] [Q&A] [answered]
Hello, 

I created a word array for my experiment & wanted to control the font size of the words in said array.  Below, I have my code as it currently is, which technically works.  

` var prime_word_list = ['<div style="font-size:55px;">Upward</div>', '<div style="font-size:55px;">Downward</div>']`

However, my output takes a hit in that it displays everything in that line and it becomes a little messy to interpret because of that.  So, I was just curious if there was a better way to create an array while also being able to control the text size without it showing in my output file?  More specifically, the only items I would want to show in my output is whether the experiment selected "Upward" or "Downward" from said array above.  

Thank you to everyone for assistance.

**Comment by becky-gilbert** [comment, marked as answer]
Hi @BobRoss048, yes you can do this by turning your [stimulus parameter into a function](https://www.jspsych.org/overview/trial/#dynamic-parameters), which will allow you to combine the static and variable parts of your stimulus HTML into one HTML string, and use that as the stimulus on each trial. 

Also, once the words are separated out from the surrounding HTML, you can save the word used on each trial in the [trial's `data` parameter](https://www.jspsych.org/overview/data/#adding-data-to-a-particular-trial-or-set-of-trials), so that it'll be easier to see the words in the data.
```javascript
var prime_word_list = ['Upward', 'Downward'];
for (var i=0; i<prime_word_list.length; i++ {
  var prime_trial = {
    type: 'html-keyboard-response',
    stimulus: function() {
      return '<div style="font-size:55px;">' + prime_word_list[i] + '</div>'
    },
    jsPsych.NO_KEYS,
    data: {word: prime_word_list[i]}
  }
  timeline.push(prime_trial);
}
```
Or if you're using [timeline variables](https://www.jspsych.org/overview/timeline/#timeline-variables) then it might look like this:
```javascript
var prime_trial_procedure = {
  timeline = [
  {
    type: 'html-keyboard-response',
    stimulus: function() {
      return '<div style="font-size:55px;">' + jsPsych.timelineVariable('word', true) + '</div>'
    },
    jsPsych.NO_KEYS,
    data: {word: jsPsych.timelineVariable('word', true)}
  },
  timeline_variables: [
    {word: 'Upward'},
    {word: 'Downward'}
  ]
};
```
